config/helm/chart/default/values.yaml

```diff
@@ -90,7 +90,7 @@ webhook:
     timeoutSeconds: 2
 
 csidriver:
-  enabled: false
+  enabled: true
```

Comment:
this would break our current release workflow, as we attach manifests without CSI Driver (see make manifests) :(

Reply:
:+1: Then, PR blocked by new user story

Reply:
@luhi-DT https://github.com/Dynatrace/dynatrace-operator/pull/2501#issuecomment-1887075087